data-model/src/main/avro/RawEventLog.avdl

```diff
@@ -0,0 +1,19 @@
+@namespace("org.hypertrace.core.datamodel")
+protocol LogEventRecordsProtocol {
```

Comment:
I'd go for `LogEvents`, is there any specific need to call it `Protocol`?

Reply:
Suffixing `Protocol` seems like a convention that we follow (just sticking to it)